config/environments/production.rb

```diff
@@ -107,9 +107,9 @@
     :authentication       => ENV['SMTP_AUTH_METHOD'] == 'none' ? nil : ENV['SMTP_AUTH_METHOD'] || :plain,
     :ca_file              => ENV['SMTP_CA_FILE'].presence || '/etc/ssl/certs/ca-certificates.crt',
     :openssl_verify_mode  => ENV['SMTP_OPENSSL_VERIFY_MODE'],
-    :enable_starttls_auto => ENV['SMTP_ENABLE_STARTTLS_AUTO'] || true,
-    :tls                  => ENV['SMTP_TLS'].presence,
-    :ssl                  => ENV['SMTP_SSL'].presence,
+    :enable_starttls_auto => ENV['SMTP_ENABLE_STARTTLS_AUTO'] != 'false',
+    :tls                  => ENV['SMTP_TLS'].presence && ENV['SMTP_TLS'] == 'true',
+    :ssl                  => ENV['SMTP_SSL'].presence && ENV['SMTP_SSL'] == 'true',
```

Comment:
Note that this can be simplified to just `ENV['SMTP_TLS'] == 'true` and `ENV['SMTP_SSL'] == 'true'`, no need for `.presence`

Reply:
I didn't know the default value for `tls` /` ssl` in ActionMailer.
Maybe it's needed for backward compatibility.

``` ruby
# if `SMTP_TLS` is unset:
ENV['SMTP_TLS'].presence # nil
ENV['SMTP_TLS'].presence && ENV['SMTP_TLS'] == 'true' # nil
ENV['SMTP_TLS'] == 'true' # false
```

Reply:
I think it behaves the same way for `nil` and `false`, though the default is indeed `nil`. I'm fine with it either way.